src/platform/silabs/multi-ota/OTATlvProcessor.h

```diff
@@ -62,12 +62,16 @@ struct OTATlvHeader
     uint32_t length;
 };
 
-// TLV tags synced with ota files generate by scripts/tools/silabs/ota/ota_image_tool.py
+// TLV tags synced with ota files generated by scripts/tools/silabs/ota/ota_image_tool.py
 enum class OTAProcessorTag
 {
     kApplicationProcessor = 1,
     kBootloaderProcessor  = 2,
-    kFactoryDataProcessor = 3
+    kFactoryDataProcessor = 3,
+    kWiFiProcessor        = 4,
+    kCustomProcessor1     = 8,
+    kCustomProcessor2     = 9,
+    kCustomProcessor3     = 10,
 };
```

Comment:
a limit or bound for the enum would be recommended for which we can test if the `OTAProcessorTag` is not unbound.

Reply:
I have added as maxValue as  11. Could you please suggest the max value for `kMaxValue`
  kMaxValue             = 11,